Dockerfile

```diff
@@ -1,39 +1,54 @@
 # Builder image that builds all the multy-back and all node services
 # multyio/multy-back-builder has all dependencies cached
+# Based on golang:1.9.4
 FROM multyio/multy-back-builder as builder
+# Get, build and install musl-gcc - a compiler that allows static builds for cgo on alpine
+RUN curl http://www.musl-libc.org/releases/musl-1.1.20.tar.gz > musl-1.1.20.tar.gz \
+    && tar -xf musl-1.1.20.tar.gz \
+    && cd ./musl-1.1.20 \
+    && ./configure --prefix=/usr && make && make install
```

Comment:
https and pipe to tar - `curl https://www.musl-libc.org/releases/musl-1.1.20.tar.gz | tar xz`